src/browser/screens/Skills.react.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import Component from 'react-pure-render/component';
+import TagsInput from 'react-tagsinput';
+
+import boundScroll from '../lib/boundScroll';
+import Chevron from '../components/Chevron.react';
+import SkillItem from '../components/SkillItem.react';
+import LeftItem from '../components/LeftItem.react';
+import technologies from '../data/technologies.json';
+
+import 'react-tagsinput/react-tagsinput.css';
+
+if (process.env.IS_BROWSER) {
+  require('./Skills.styl');
+}
+
+export default class Skills extends Component {
+  static propTypes = {
+    actions: React.PropTypes.object.isRequired,
+    candidate: React.PropTypes.object.isRequired
+  }
+
+  constructor() {
+    super()
+    this.state = {skills: []}
+    this.handleChange = this.handleChange.bind(this);
```

Comment:
wczesniej robiles w momencie wywolania `() => this.handleChange()`, wiec dobrze trzymac sie tej konwencji

Reply:
Funkcja odwołuje się do libki więc w tym wypadku kontekst przy takim wywołaniu byłby inny.

Reply:
Wydaje mi sie ze kontekst bedzie ten sam. Tutaj przekazujesz kontekst `this` i arrow zrobi to samo z automatu.